Makefile

```diff
@@ -236,6 +242,11 @@ controller-gen: $(CONTROLLER_GEN) ## Download controller-gen locally if necessar
 $(CONTROLLER_GEN): $(LOCALBIN)
 	test -s $(CONTROLLER_GEN) || GOBIN=$(LOCALBIN) go install sigs.k8s.io/controller-tools/cmd/controller-gen@$(CONTROLLER_GEN_VERSION)
 
+.PHONY: yq
+yq: $(YQ) ## Download controller-gen locally if necessary.
```

Comment:
Probably, not controller-gen.

Reply:
Sloppy me. Copy-pastie design pattern. Fixed in df5a052